zoobot/pytorch/examples/train_model_on_catalog.py

```diff
@@ -22,14 +22,18 @@
     parser.add_argument('--experiment-dir', dest='save_dir', type=str)
     # expects catalog, not tfrecords
     parser.add_argument('--catalog', dest='catalog_loc', type=str)
-    parser.add_argument('--architecture', dest='model_architecture', type=str)
+    parser.add_argument('--num_data_workers',
+                        dest='num_workers', type=int, default=int((os.cpu_count() * 2)))
+    parser.add_argument('--architecture',
+                        dest='model_architecture', type=str, default='efficientnet')
     parser.add_argument('--epochs', dest='epochs', type=int)
     parser.add_argument('--shard-img-size',
                         dest='shard_img_size', type=int, default=300)
     parser.add_argument('--resize-size', dest='resize_size',
                         type=int, default=224)
     parser.add_argument('--batch-size', dest='batch_size',
                         default=256, type=int)
+    parser.add_argument('--accelerator', type=str, default='gpu')
```

Comment:
Notes for self

num_data_workers -> num_workers
check effect of auto accel arg here